README.rst

```diff
@@ -18,24 +18,22 @@ cssselect: CSS Selectors for Python
    :target: https://codecov.io/github/scrapy/cssselect?branch=master
    :alt: Coverage report
 
-*cssselect* parses `CSS3 Selectors`_ and translate them to `XPath 1.0`_
-expressions. Such expressions can be used in lxml_ or another XPath engine
-to find the matching elements in an XML or HTML document.
+**cssselect** is a BSD-licensed Python library to parse `CSS3 selectors`_ and
+translate them to `XPath 1.0`_ expressions.
 
-This module used to live inside of lxml as ``lxml.cssselect`` before it was
-extracted as a stand-alone project.
-
-.. _CSS3 Selectors: https://www.w3.org/TR/css3-selectors/
-.. _XPath 1.0: https://www.w3.org/TR/xpath/
-.. _lxml: http://lxml.de/
+`XPath 1.0`_ expressions can be used in lxml_ or another XPath engine to find
+the matching elements in an XML or HTML document.
 
+Find the cssselect online documentation at https://cssselect.readthedocs.io.
 
 Quick facts:
 
-* Free software: BSD licensed
-* Compatible with Python 2.7 and 3.4+
-* Latest documentation `on Read the Docs <https://cssselect.readthedocs.io/>`_
 * Source, issues and pull requests `on GitHub
   <https://github.com/scrapy/cssselect>`_
 * Releases `on PyPI <http://pypi.python.org/pypi/cssselect>`_
 * Install with ``pip install cssselect``
+
+
+.. _CSS3 selectors: https://www.w3.org/TR/css3-selectors/
+.. _XPath 1.0: https://www.w3.org/TR/xpath/
+.. _lxml: http://lxml.de/
```

Comment:
Some links that changed with permanent redirects, since I just ran across those for scrapy docs:

`on PyPI <http://pypi.python.org/pypi/cssselect>`_ => `on PyPI <https://pypi.org/project/cssselect/>`_
https://www.w3.org/TR/css3-selectors/ => `https://www.w3.org/TR/selectors-3/`
https://www.w3.org/TR/xpath/ => `https://www.w3.org/TR/xpath/all/`
http://lxml.de/ => `https://lxml.de/`

I just double-checked them, so you may copy&paste them from here.